2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_120206-34s9vyrl/logs/debug.log
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_120206-34s9vyrl/logs/debug-internal.log
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_init.py:init():404] calling init triggers
2022-01-14 12:02:06,988 INFO    MainThread:340113 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 12:02:06,989 INFO    MainThread:340113 [wandb_init.py:init():460] starting backend
2022-01-14 12:02:06,989 INFO    MainThread:340113 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 12:02:06,995 INFO    MainThread:340113 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 12:02:06,998 INFO    MainThread:340113 [backend.py:ensure_launched():221] started backend process with pid: 340172
2022-01-14 12:02:06,999 INFO    MainThread:340113 [wandb_init.py:init():469] backend started and connected
2022-01-14 12:02:07,002 INFO    MainThread:340113 [wandb_init.py:init():533] updated telemetry
2022-01-14 12:02:07,011 INFO    MainThread:340113 [wandb_init.py:init():563] communicating current version
2022-01-14 12:02:07,395 INFO    MainThread:340113 [wandb_init.py:init():568] got version response 
2022-01-14 12:02:07,396 INFO    MainThread:340113 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 12:02:07,593 INFO    MainThread:340113 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 12:02:07,645 INFO    MainThread:340113 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 12:02:07,645 INFO    MainThread:340113 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 12:02:07,645 INFO    MainThread:340113 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 12:02:07,646 INFO    MainThread:340113 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 12:02:07,646 INFO    MainThread:340113 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 12:02:07,872 INFO    MainThread:340113 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 12:02:07,873 INFO    MainThread:340113 [wandb_run.py:_restore():1752] restore
2022-01-14 12:02:10,157 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 12:02:10,332 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 12:02:10,600 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 2141
}

2022-01-14 12:02:10,701 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4114
}

2022-01-14 12:02:10,802 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2778
  total_bytes: 4114
}

2022-01-14 12:02:10,903 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:11,007 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:11,109 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:11,212 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:11,316 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:11,725 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}

2022-01-14 12:02:12,107 INFO    MainThread:340113 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4114
  total_bytes: 4114
}
local_info {
}

2022-01-14 12:02:13,170 INFO    MainThread:340113 [wandb_run.py:_append_files():2180] logging synced files
